doc.ddoc

```diff
@@ -1,3 +1,4 @@
+ROOT_DIR =
```

Comment:
OK I'm really confused, is the consensus that we want three macros for the same thing? I can add `ROOT_SLASH` but I see no practical point in it. Should it replace one of the two macros? Which one?